File: 23-09-2026/lab9/q2.py
def daysElapsed(day, month):
    months = [31,28,31,30,31,30,31,31,30,31,30,31]
    dE = sum(months[:month - 1])
    dE += day
    return dE

day, month = map(int, input().split())
print(daysElapsed(day, month))

File: 23-09-2026/lab9/q2.c
#include<stdio.h>

int daysElapsed(int day,int month){
	int months[12]={31,28,31,30,31,30,31,31,30,31,30,31};
	int dE = 0;
	if(day<=months[month-1]){if(month>1){
	for(int i=0;i<month-1;i++){
		dE += months[i];
	}}
	
	dE += day;}
	return dE;
}

int main(){
	int day, month;
	scanf("%d %d", &day,&month);
	printf("%d",daysElapsed(day, month));
	return 0;
}

